tests/utils.go

```diff
@@ -385,3 +388,36 @@ func GenerateRandomSQLInsert(maxLength int) (string, error) {
 
 	return query, nil
 }
+
+func WaitForMsg(t *testing.T, wg *sync.WaitGroup, ch chan *protocol.Envelope) {
+	wg.Add(1)
+	log := utils.Logger()
+	go func() {
+		defer wg.Done()
+		select {
+		case env := <-ch:
+			msg := env.Message()
+			log.Info("Received ", zap.String("msg", msg.String()))
+		case <-time.After(2 * time.Second):
+			require.Fail(t, "Message timeout")
+		}
+	}()
+	wg.Wait()
+}
+
+func WaitForTimeout(t *testing.T, ctx context.Context, wg *sync.WaitGroup, ch chan *protocol.Envelope) {
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		select {
+		case _, ok := <-ch:
+			require.False(t, ok, "should not retrieve message")
+		case <-time.After(1 * time.Second):
```

Comment:
minor suggest, maybe take timeout as an arg to the method so that it can be used in other tests as well.

Reply:
Thanks for suggestion. Added timeout param to both WaitFor funcs. https://github.com/waku-org/go-waku/pull/1060/commits/99e3f334cd8356d343d3f3f7ac750ca46508de05